README.md

```diff
@@ -12,6 +12,9 @@ $ yarn add @react-native-community/async-storage
 
 # Link
 $ react-native link @react-native-community/async-storage
+
+# Pod installation
+$ cd ios/ && pod install
```

Comment:
Can you move this bit below (just above "See docs...") and note it's only for `iOS` project with `pods`?

Something like:
```markdown
**Note:** For iOS project using `pods`, run:
	```
	$ cd ios/ && pod install
	```
```

Reply:
Done, also!